test/init-mem.c

```diff
@@ -98,9 +98,9 @@ static int test(struct q_entries *q)
 	if (ret == T_EXIT_SKIP) {
 		clean_ctx(&ctx);
 		return T_EXIT_SKIP;
-	}
-	else if (ret != T_EXIT_PASS)
```

Comment:
Same, unrelated style change.

Reply:
This was in response to an earlier comment you made, about if the if needs braces, the else if/else does too.

Reply:
Moved change to previous commit